packages/wdio-cucumber-framework/src/index.js

```diff
@@ -89,7 +94,8 @@ class CucumberAdapter {
              */
             setUserHookNames(Cucumber.supportCodeLibraryBuilder.options)
             Cucumber.setDefaultTimeout(this.cucumberOpts.timeout)
-            const supportCodeLibrary = Cucumber.supportCodeLibraryBuilder.finalize()
+            global.supportCodeLibrary = Cucumber.supportCodeLibraryBuilder.finalize()
+            }
```

Comment:
Lines 81-97's scope is missing indentation.

Reply:
See comment above ;)